parsl/monitoring/visualization/templates/files.html

```diff
@@ -0,0 +1,48 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<br />
+    <h2>File Provenance</h2>
+
+     Files by <a href="/">workflow</a>.
+    <p>
+    Search for a specific file (use % as a wildcard):
```

Comment:
what's being searched here? the filename?

Reply:
Yes, I have updated the text to make it more clear.